explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -231,14 +265,7 @@ function AccountsCard({ signature }: Props) {
 
   if (!status?.data?.info) {
     return null;
-  } else if (!details) {
-    return (
-      <ErrorCard
-        retry={refreshStatus}
-        text="Details are not available until the transaction reaches MAX confirmations"
-      />
-    );
-  } else if (details.status === FetchStatus.Fetching) {
+  } else if (!details || details.status === FetchStatus.Fetching || status.data.info.confirmations !== "max") {
```

Comment:
Cool, so this will now assume that auto refresh is always happening? This is fine with me, but another thing you could do that you sorted hinted at is pulling the `autoRefreshInProcess` state up a level. Wouldn't need to pull it into a context provider, but it could be pulled up into the `TransactionDetailsPage` component. That way it could be set in `StatusCard` and could be passed down via props to be used in `AccountsCard`

Reply:
Good call.